app/jobs/email_csv_job.rb

```diff
@@ -0,0 +1,21 @@
+class EmailCsvJob < ApplicationJob
+  queue_as :default
+
+  BYTE_ORDER_MARK = "\uFEFF".freeze # Required to ensure Excel always reads CSV as UTF-8
+
+  EXPIRATION_TIME = 3.hours.to_i
+
+  def perform(user, search_term = nil, filters = {}, all_orgs = false, organisation = nil) # rubocop:disable Style/OptionalBooleanParameter
```

Comment:
Any reason not to just make these keyword args rather than turning off the cop?

Reply:
Sidekiq doesn't support serialising keyword args. But I should probably add a comment to that effect 🙂